app/models/planet.py

```diff
@@ -0,0 +1,23 @@
+from app import db
+
+class Planet(db.Model):
+    id=db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name=db.Column(db.String, nullable=False)
+    description=db.Column(db.String, nullable=False)
+    solar_day=db.Column(db.Float, nullable=False)
```

Comment:
Good idea to make these attributes non-nullable! That being said, it can be beneficial to have some nullable attributes. It all boils down to how you want to design your particular class though!